src/software/SfM/main_ConvertSfM_DataFormat.cpp

```diff
@@ -53,24 +53,21 @@ void updateStructureWithNewUID(Landmarks &landmarks, const std::map<std::size_t,
 }
 
 /**
- * @brief Recompute the UID from the metadata of the original input images and 
- * modify the ID if it's not the same.
+ * @brief Update all viewID of a list of view by replacing them with the UID.
```

Comment:
"by replacing them with the UID" -> not true, this function only remap the ID you give in input with oldIdToNew.